src/main/java/com/cloudogu/scm/editor/EditorService.java

```diff
@@ -134,6 +137,14 @@ public FileUploader modify(String fileName, InputStream stream) {
     }
 
     private String computeCompleteFileName(String fileName) {
```

Comment:
I think we should do two things:
1. Disallow a leading `/` in the core modify command
2. Remove leading `/` in this method, so that the user can still enter it (may be intuitive for the user to say that the path should start at the root of the repository)